terraform-module/modules/frontend-spa-dns/main.tf

```diff
@@ -38,3 +38,13 @@ resource "aws_route53_record" "dns_record" {
     evaluate_target_health = false
   }
 }
+
+// Request for app.demo.pleo.io from ask-SRE ticket SRE-7285
+resource "aws_route53_record" "demo_dns_record" {
```

Comment:
spa-tools is a generic terraform-module used by multiple web-apps, I don't think it should have this record that is specific to product-web. I'm also not sure if this would work in practice since the same record would be created for each consumer of spa-tools, for example backoffice and the telescope docs.

I think we should move this to the product-web component in terraform instead.

Reply:
nevermind, already addressed by matej's changes